docs/changelog-fragments.d/233.bugfix.md

```diff
@@ -0,0 +1,2 @@
+Fixed source of INVALID_URL type error that occurred during the search for
```

Comment:
Since this is an identifier, let's use Markdown syntax to show that.
```suggestion
Fixed source of `INVALID_URL` type error that occurred during the search for
```

Reply:
Thanks for the suggestion. Updated !